packages/table/src/presenters/TablePresenter.js

```diff
@@ -278,7 +278,8 @@ const TablePresenter = ({
                         ? stylesheet(
                             { isResizing: column.isResizing },
                             resolvedRoles,
-                            metadata
+                            metadata,
+                            customStylesheet
```

Comment:
should move the `customStylesheet` prop into the object like so:
`{ isResizing: column.isResizing, customStylesheet}`